frontend/src/features/properties/map/research/add/AddResearchContainer.tsx

```diff
@@ -25,6 +25,7 @@ export const AddResearchContainer: React.FunctionComponent<IAddResearchContainer
   const saveResearchFile = async (researchFile: Api_ResearchFile) => {
     const response = await addResearchFile(researchFile);
     if (!!response?.name) {
+      formikRef.current?.resetForm();
```

Comment:
I'm not so sure about this, Do we really want the behaviour of an error to be wiping the form? what if the error is correctible on the frontend? what if the error is due to a network timeout and just requires the form to be resubmitted?